ethers-core/src/types/chain.rs

```diff
@@ -502,6 +506,9 @@ impl Chain {
 
             ZkSync => {
                 ("https://zksync2-mainnet-explorer.zksync.io/", "https://explorer.zksync.io/")
+            },
+            ZkSyncTestnet => {
+                ("https://zksync2-testnet-explorer.zksync.dev/", "https://goerli.explorer.zksync.io/")
```

Comment:
I added the explorer api (the first element of the tuple) for ZkSync testnet, because it's there for mainnet as well. But as far as a I can tell, the explorer API is not compatible with Etherescan, it only seems to support [contract verification.](https://era.zksync.io/docs/api/hardhat/hardhat-zksync-verify.html#configuration)